pkg/httpd/server.go

```diff
@@ -0,0 +1,95 @@
+package httpd
+
+import (
+	"context"
+	"errors"
+	"net"
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+var ShutdownTimeout = time.Second * 5
```

Comment:
This can be a const.

Reply:
On a test I rely on being able to change this. Generally I don't think it's bad to have a knob to change this.

Reply:
👍 I didn't notice the test case usage before now.